spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -181,20 +187,21 @@ public class SampleTask {
 
 		@Override
 		public void run(String... strings) throws Exception {
-			System.out.println("Hello World!");
+			System.out.println("Hello, World!");
 		}
 	}
 }
 ----
 
-While it may not look like much, quite a bit is going on.  To read more about the Spring
-Boot specifics, take a look at their reference documentation here:
-http://docs.spring.io/spring-boot/docs/current/reference/html/[http://docs.spring.io/spring-boot/docs/current/reference/html/]
+While it may seem small, quite a bit is going on. For more about Spring
+Boot specifics, see the
+http://docs.spring.io/spring-boot/docs/current/reference/html/[Spring Boot reference documentation].
 
-We'll also need to create an `application.properties` in `src/main/resources`.  We'll
-configure two properties in it: the application name (which is translated to the task name)
-and we'll set the logging for spring cloud task to `DEBUG` so that we can see what's going
-on:
+We also need to create an `application.properties` file in `src/main/resources`. We need
+to configure two properties in `application.properties`: We need to set the application
+name (which is translated to the task name), and we need to set the logging for spring
```

Comment:
Spring Cloud Task (caps) just for consistency.

Reply:
Yup